### PR TITLE
en: Add description about continue profile

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Add description about continue profile

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/1581, https://github.com/pingcap/docs-tidb-operator/pull/1585, https://github.com/pingcap/docs-tidb-operator/pull/1595
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/access-dashboard.md [hunk 1]
```diff
@@ -204,6 +204,56 @@ After deploying the `Service`, you can access TiDB Dashboard via <https://{nodeI
 
 Note that if there is more than one PD `Pod` in the cluster, you need to set `spec.pd.enableDashboardInternalProxy: true` in the `TidbCluster` CR to ensure normal access to TiDB Dashboard.
 
+## Enable Continuous Profiling
+
+With Continuous Profiling, you can collect continuous performance data of TiDB, PD, TiKV, and TiFlash instances, and have the nodes monitored day and night without restarting any of them. The data collected can be displayed in various forms, for example, on a flame graph or a directed acyclic graph. The data displayed visually shows what internal operations are performed on the instances during the performance profiling period and the corresponding proportions. With such data, you can quickly learn the CPU resource consumption of these instances.
+
+To enable this feature, you need to deploy TidbNGMonitoring CR using TiUP v1.3.0 or later.
```
Comment (KanShiori) [inline en/access-dashboard.md:211]:
```suggestion
To enable this feature, you need to deploy TidbNGMonitoring CR using TiDB Operator v1.3.0 or later.
```

**Review by KanShiori — APPROVED** [review]

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**